tests/d3d12.c

```diff
@@ -48996,6 +48996,118 @@ static void test_cbv_hoisting_dxil(void)
     test_cbv_hoisting(true);
 }
 
+static void test_write_watch(void)
+{
+#ifndef _WIN32
+    skip("WRITE_WATCH tests cannot pass on native Linux. Skipping.\n");
+#else
+    D3D12_HEAP_PROPERTIES heap_properties;
+    D3D12_RESOURCE_DESC resource_desc;
+    struct test_context_desc desc;
+    void **dirty_addresses = NULL;
+    struct test_context context;
+    ULONG_PTR address_count;
+    ID3D12Resource *buffer;
+    size_t mapping_size;
+    DWORD page_size;
+    char *map_ptr;
+    UINT result;
+    HRESULT hr;
+
+    mapping_size = 64 * 1024;
+
+    memset(&desc, 0, sizeof(desc));
+    desc.no_render_target = true;
+    desc.no_pipeline = true;
+    desc.no_root_signature = true;
+    if (!init_test_context(&context, &desc))
+        return;
+
+    memset(&heap_properties, 0, sizeof(heap_properties));
+
+    resource_desc.Dimension = D3D12_RESOURCE_DIMENSION_BUFFER;
+    resource_desc.Alignment = 0;
```

Comment:
Is this okay?

Reply:
Alignment = 0? Yeah.

`Alignment must be 64KB (D3D12_DEFAULT_RESOURCE_PLACEMENT_ALIGNMENT) or 0, which is effectively 64KB.`